zipline/pipeline/graph.py

```diff
@@ -70,7 +70,9 @@ def offset(self):
 
             Max number of extra rows needed by any term depending on `input`
             minus
-            Number of extra rows needed by `term`.
+            Number of rows needed by `input` for `term`
```

Comment:
This is necessary to accommodate passing `Returns`, a windowed term, as an input to another factor.